Sources/Utils/Dropdown/DropdownController.swift

```diff
@@ -11,7 +11,7 @@ class DropdownController: UIViewController {
   lazy var blurView: UIVisualEffectView = self.makeBlurView()
 
   var animating: Bool = false
-  var expanding: Bool = true
+  var expanding: Bool = false
```

Comment:
Why did you change the default here? Will this have an impact on existing projects?

Reply:
In the previous commit this flag was set to true, and after the dropdownController.toggle () method was called. Just a little optimization. Current projects are not affected in any way

Reply:
Nice, just wanted to make sure 🚀